src/component/AuthForms/LoginForm.jsx

```diff
@@ -74,7 +107,7 @@ const LoginForm = () => {
 
                 <div className="input-row">
                     <label className="input-label">
-                        <input type="checkbox" className="input checkbox-input" />
+                        <input type="checkbox" className="input checkbox-input" required />
                         <span className="checkbox-text">Remember me</span>
```

Comment:
The Remember Me checkbox should not be given the `required` attribute. The user might not want to check it.